tests/e2e/testsuites/format_options_tester.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testsuites
+
+import (
+	awscloud "github.com/kubernetes-sigs/aws-ebs-csi-driver/pkg/cloud"
+	"github.com/kubernetes-sigs/aws-ebs-csi-driver/tests/e2e/driver"
+	. "github.com/onsi/ginkgo/v2"
+	v1 "k8s.io/api/core/v1"
+	clientset "k8s.io/client-go/kubernetes"
+)
+
+// FormatOptionTest will provision required StorageClass(es), PVC(s) and Pod(s) in order to test that volumes with
+// a specified custom format options will mount and are able to be later resized.
+type FormatOptionTest struct {
+	CreateVolumeParameterKey   string
+	CreateVolumeParameterValue string
+}
+
+const (
+	volumeSizeIncreaseAmtGi = 1
+)
+
+func (t *FormatOptionTest) Run(client clientset.Interface, namespace *v1.Namespace, ebsDriver driver.PVTestDriver, fsType string) {
+	By("setting up pvc with custom format option")
+	volumeDetails := createFormatOptionVolumeDetails(fsType, t)
+	testPvc, _ := volumeDetails.SetupDynamicPersistentVolumeClaim(client, namespace, ebsDriver)
+	defer testPvc.Cleanup()
+
+	By("deploying pod with custom format option")
+	getFsInfoTestPod := createPodWithVolume(client, namespace, PodCmdWriteToVolume(DefaultMountPath), testPvc, volumeDetails)
+	defer getFsInfoTestPod.Cleanup()
+	getFsInfoTestPod.WaitForSuccess()
```

Comment:
How does the `getFsInfoTestPod` actually get fs info? It doesn't even seem to be created with the necessary privileges to do that?